docs/experiments-plus/sequential-testing.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Sequential Testing
+sidebar_label: Sequential Testing
+slug: /experiments-plus/sequential-testing
+---
+
+## What is Sequential Testing?
+
+Traditional A/B testing best practices dictate that the readout of experiment metrics should occur only once, when the target sample size of the experiment has been reached.  Continous monitoring for the purpose of decision making results in inflated false positive rates, much higher than expected based on the significance level selected for the test.  
```

Comment:
Same feedback.  I feel like most folks know this as "peeking" or the "peeking problem".